### ISSUE TITLE
Url with og:description set returns incorrect description

### ISSUE THREAD

**Issue by kivvii** [jaimeiniesta/metainspector#181]
For the url http://winsupersite.com/windows-10/prevent-windows-10-upgrade-installing-after-making-your-reservation there is a valid `og:description`: 

```
"If you are not ready for the Windows 10 upgrade but made a reservation these steps will help you keep it off your system."
```

but fetching `description` returns a chunk of javascript: 

```
"')})();\n//-->\n\n\n<!--//--><![CDATA[//><!--\ngoogletag.defineSlot('/3834/wss.home/windows/windows_10',[[180,150]],'gpt-180_1').addService(googletag.pubads()).setTargeting('pos','180_1');googletag.defineSlot('/3834/wss.home/windows/windows_10',[[300,100]],'gpt-...
```

**Comment by jschwindt** [comment]
There is no meta description header, so it's trying to take incorrectly the first paragraph. I think we should create a method `best_desciption` similar to `best_title`, so we can prioritize "og:description".

**Comment by jaimeiniesta** [comment]
@kivvii thanks for submitting this issue, there is clearly a bug because `description` should be returning currently the `meta name="description"` tag and, if not available as in this case, it should the first paragraph which is at least 120 characters. But it's getting a script from the head instead, we need to investigate why and fix that.

@jschwindt I agree, we can make `description` return only the meta tag if it exists, or nil if it doesn't, and then have `best_description` that would prioritize `og:description` over `secondary_description`.

**Comment by prabond** [comment]
Any update on this issue?

**Comment by jaimeiniesta** [comment]
@prabond yes, there's a PR for fixing this and we're discussing it https://github.com/jaimeiniesta/metainspector/pull/197

**Comment by jaimeiniesta** [comment]
Fixed, in MetaInspector 5.4.1 this URL now returns a `best_description`:

```ruby
page.best_description
 => "We have had numerous questions from users on Windows 7 and 8.1 systems who, after reserving their free Windows 10 upgrade using the Get Windows 10 App, have decided to not upgrade."
```
